jbdevstudio/com.jboss.jbds.central.discovery.earlyaccess/plugin.xml

```diff
@@ -75,5 +65,30 @@ This content is unavailable. Enable Early Access to see it.</description>
          <overview url="http://www.jboss.org/tools"/>
       </connectorDescriptor>
 
+       <!-- JBoss Fuse Development -->
```

Comment:
eh - I know this is for testing purposes but why add this to the main descriptor and not to the IS files where this is suppsoed to be ?